satpy/readers/seviri_base.py

```diff
@@ -373,7 +374,8 @@ def _tl15(self, data, wavenumber):
 
     def _vis_calibrate(self, data, solar_irradiance):
         """Calibrate to reflectance."""
```

Comment:
Maybe add a reference here? E.g.

http://www2.geog.ucl.ac.uk/~mdisney/teaching/PPRS/papers/schaepman_et_al.pdf
or
http://glossary.ametsoc.org/wiki/Bidirectional_reflectance_factor

Reply:
I've added a reference to the corresponding EUMETSAT doc (Conversion from radiances to reflectances for SEVIRI warm channels).